docs/build-on-linea/tooling/attestations/attestations.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Attestations
+---
+
+An attestation is evidence or proof of something. Some real world examples would be:
```

Comment:
Change second sentence to: "Real world examples of attestations are:"

Reply:
fixed